### ISSUE TITLE
Use rows library to convert values

### ISSUE THREAD

**Issue by turicas** [datasciencebr/jarbas#211]
jarbas could use some library to automatically identify and convert data types (from strings) instead of doing it itself (like on this [serialize](https://github.com/datasciencebr/jarbas/blob/master/jarbas/core/management/commands/companies.py#L66) method). The [rows library](http://turicas.info/rows) does it automatically and will probably remove a lot of jarbas code if used here, so the import management commands will be easier to write and maintain.

This issue strongly related to https://github.com/datasciencebr/serenata-toolbox/issues/101

cc @cuducos

**Comment by cuducos** [comment]
I think it's a great idea to use `rows` to better handle data input here in Jarbas!

**Comment by cuducos** [comment]
_[Just pasting From (closed) #198]_

[`rows`](http://turicas.info/rows/) already deals with a lot of common issues related to reading a CSV file and serializing data. With that in mind maybe we ca reduce the LOC to maintain in this project if we use `rows` to deal with:

- [ ] [importing and serializing reimbursement data](https://github.com/datasciencebr/jarbas/blob/master/jarbas/core/management/commands/reimbursements.py#L35-L82) (basically handling [this transformation](https://github.com/datasciencebr/jarbas/blob/master/jarbas/core/tests/test_reimbursements_command.py#L20-L89))
- [ ] [importing and serializing suspicions data](https://github.com/datasciencebr/jarbas/blob/master/jarbas/core/management/commands/irregularities.py#L37-L71) (basically handling [this transformation](https://github.com/datasciencebr/jarbas/blob/master/jarbas/core/tests/test_irregularities_command.py#L18-L35))
- [ ] [importing and serializing companies data](https://github.com/datasciencebr/jarbas/blob/master/jarbas/core/management/commands/companies.py#L34-L77) (basically handling [this transformation](https://github.com/datasciencebr/jarbas/blob/master/jarbas/core/tests/test_companies_command.py#L20-L42) and paying attention to the creation of `Activity` data)

**Comment by Ilozuluchris** [comment]
@cuducos,I think I can do this.

**Comment by cuducos** [comment]
Sounds great @Ilozuluchris — I got started one of these days (check #242), so you can either:

* Cover the scripts I haven't covered yet (if I remember right, suspicions and company importion)
* Continue from where I left opening a PR to my branch
* Throw away my code and do it all by yourself

Third option sounds like exaggerating, but might make sense in one or another scenario. Anyway I strongly recommend you to follow the advices @turicas left as a code review to my WIP PR over there ; )

**Comment by annacruz** [comment]
This issue still need to be done?

**Comment by cuducos** [comment]
Yep… there’s two WIP but still at least one case to be covered: take a look in the discussion and drafts on #242 and #262 and count on me ; )
